src/sentry/services/hybrid_cloud/auth/impl.py

```diff
@@ -153,6 +146,14 @@ def get_org_auth_config(
             for oid in organization_ids
         ]
 
+    def _serialize_auth_user(self, user: User) -> RpcUser | None:
+        rpc_user: RpcUser | None = None
+        if user is not None:
+            users = user_service.get_many(filter=dict(user_ids=[user.id]))
+            if users:
+                rpc_user = users[0]
+        return rpc_user
```

Comment:
I don't think this method's name makes sense. It isn't serializing the state of the argument; it's retrieving the remote state of the user with the argument's ID and ignoring everything else about it. Is there some new state that you're trying to retrieve? If so, it would be helpful to have a comment explaining it.

Also, I believe you could just call `user_service.get_user(user.id) if user else None` and get the same result.